cypher/cypher-docs/src/docs/dev/ql/administration/security/grant-create-syntax.asciidoc

```diff
@@ -1,11 +0,0 @@
-.Command syntax
```

Comment:
Content was moved to neo4j-manual-modeling/4.3/cypherManual/asciidoc/access-control/privileges-write.adoc

https://neo4j.com/docs/cypher-manual/4.3/access-control/privileges-writes/#access-control-privileges-writes-create